assets/js/components/DashboardNavigation.js

```diff
@@ -30,14 +30,69 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import Data from 'googlesitekit-data';
 import {
 	ANCHOR_ID_CONTENT,
 	ANCHOR_ID_MONETIZATION,
 	ANCHOR_ID_SPEED,
 	ANCHOR_ID_TRAFFIC,
 } from '../googlesitekit/constants';
+import { CORE_WIDGETS } from '../googlesitekit/widgets/datastore/constants';
+import {
+	CONTEXT_ENTITY_DASHBOARD_TRAFFIC,
+	CONTEXT_ENTITY_DASHBOARD_CONTENT,
+	CONTEXT_ENTITY_DASHBOARD_SPEED,
+	CONTEXT_ENTITY_DASHBOARD_MONETIZATION,
+	CONTEXT_MAIN_DASHBOARD_TRAFFIC,
+	CONTEXT_MAIN_DASHBOARD_CONTENT,
+	CONTEXT_MAIN_DASHBOARD_SPEED,
+	CONTEXT_MAIN_DASHBOARD_MONETIZATION,
+} from '../googlesitekit/widgets/default-contexts';
+import useDashboardType, {
+	DASHBOARD_TYPE_MAIN,
+} from '../hooks/useDashboardType';
+
+const { useSelect } = Data;
 
 export default function DashboardNavigation() {
+	const dashboardType = useDashboardType();
+
+	const [ showTraffic, showContent, showSpeed, showMonitization ] = useSelect(
```

Comment:
Thanks, @kuasha420. This works as expected, but we don't encourage useSelect hooks with multiple selects (although we have some exception in the codebase). Could you, please, split it into four useSelect hooks for each individual chips?

```js
const showTraffic = useSelect( ( select ) => select( CORE_WIDGETS ).isWidgetContextActive(
    dashboardType === DASHBOARD_TYPE_MAIN
        ? CONTEXT_MAIN_DASHBOARD_TRAFFIC
        : CONTEXT_ENTITY_DASHBOARD_TRAFFIC
) );
```

Reply:
Done, Is there any reason for not reusing the useSelect call? Is there any correctness issue here? Using multiple select for same store seems kinda odd and verbose at first glance.

Reply:
There is nothing wrong with using multiple selects in the same hook. However, we have a convention to use a separate hook for each select because it makes it easier to read the code. If you think this should be changed, feel free to raise this question during our engineering sync call and we can discuss it.